apps/server/src/modules/deletion/domain/service/deletion-execution.service.ts

```diff
@@ -35,6 +39,13 @@ export class DeletionExecutionService {
 			await this.deletionRequestService.markDeletionRequestAsExecuted(deletionRequest.id);
 		} catch (error) {
 			await this.deletionRequestService.markDeletionRequestAsFailed(deletionRequest.id);
+
+			this.logger.warning(
+				new DeletionErrorLoggableException(
+					`An error occurred during deletion execution ${deletionRequest.id}`,
+					error as Error
+				)
+			);
```

Comment:
Okay, this is in case of any unexpected error during the execution of the deletion request right? But we're still marking the deletion request as failed with `deletionRequestService.markDeletionRequestAsFailed(deletionRequest.id)`. Is this to have some additional information in the logs?